topi/python/topi/cpp/vision/__init__.py

```diff
@@ -0,0 +1,7 @@
+"""FFI for vision TOPI ops and schedules"""
```

Comment:
Why this file does not have an ASF header and could still pass our test?

Reply:
I do not know. When linting, it just asks me to add ASF header to new files but `init.py`.

Reply:
https://github.com/apache/incubator-tvm/blob/master/topi/python/topi/x86/__init__.py
This also does not have ASF header.
I do not know why either.

Reply:
Okay, then I am fine with it :-)